### PR TITLE
Add monster can open sound effect for reload, deploy, & init

### PR DESCRIPTION
**Pull request by Narshalbarsh**
Improves boomer monster immersion, note that the can opening sound effect was sourced from an actual white monster can being opened for maximum realism

### CODE REVIEW

File: lua/cinema/weapons/weapon_monster/shared.lua [hunk 1]
```diff
@@ -22,6 +22,19 @@ function SWEP:Initialize()
     self:SetHoldType("slam")
 end
 
+function SWEP:Deploy()
+    BaseClass.Deploy(self)
+    local owner = self:GetOwner()
+
+    if IsValid(owner) then
```
Comment (swampservers) [inline lua/cinema/weapons/weapon_monster/shared.lua]: unnecessary check, remove the baseclass.deploy also

**Comment by swampservers** [comment]
added dev tools access, thanks!

**Comment by Kody-Quintana** [comment]
@swampservers thanks for the dev tool access, I have this "working" locally but only with `EmitSound` and not `ExtEmitSound`.

`ExtEmitSound` works with all the other sound files, just not the new one. Does `ExtEmitSound` not work with new files that aren't actually on the server yet?

**Comment by Narshalbarsh** [comment]
@swampservers thanks for the dev tool access. I have this "working" locally, but only with `EmitSound` and not `ExtEmitSound`.

`ExtEmitSound` works with all the other sound files, just not the new one. Does `ExtEmitSound` not work with new files that aren't actually on the server yet?

**Review by WinterPhoenix — APPROVED** [review]
We already talked about this elsewhere, but `ExtEmitSound` is deprecated and only really useful for lip-syncing. Looks good to me

**Comment by Narshalbarsh** [comment]
Thank you winter, I'm already feeling more immersed

**Comment by swampservers** [comment]
> We already talked about this elsewhere, but `ExtEmitSound` is deprecated and only really useful for lip-syncing. Looks good to me

It also handles locations (so you dont hear everything from inside a theater) and the crouch pitch shifting. Someone is already complaining about it being audible map wide

**Comment by Narshalbarsh** [comment]
> It also handles locations (so you dont hear everything from inside a theater) and the crouch pitch shifting. Someone is already complaining about it being audible map wide

Seems like `ExtEmitSound` not working was only because the sound file was new, so the live preview of the dev tools couldn't use it?

Now that the wav file is merged, it works. Here's a PR https://github.com/swampservers/contrib/pull/465/files

**Comment by WinterPhoenix** [comment]
The `EntityEmitSound` hook/network handling we have is supposed to deal with that regardless.

**Comment by swampservers** [comment]
> The `EntityEmitSound` hook/network handling we have is supposed to deal with that regardless.

No, that is only for global volume control. The problem with EmitSound hooks is we don't know what the sound came from beyond it being a player. The ExtEmitSound makes it so when you are in a theater you can't hear soundplayer sweps from outside the theater regardless of your volume settings. Also the crouch pitch shifting is included in there.

**Pull request merged**